modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,74 @@
+
+The StackRox Scanner originates from a fork of the Clair v2 open source scanner and is the default scanner. The StackRox Scanner is deprecated beginning with release 4.6. In version 4.4, {product-title-short} introduced Scanner V4, built on ClairCore, which provides additional image scanning features.
+
+[id="stackrox-scanner-vuln-sources"]
+== StackRox Scanner sources
+
+The StackRox Scanner uses the following vulnerability sources:
+
+* https://access.redhat.com/security/data/oval/v2/[Red{nbsp}Hat OVAL] v2
+* link:https://secdb.alpinelinux.org/[Alpine Security Database]
+* Data tracked in link:https://alas.aws.amazon.com/index.html[Amazon Linux Security Center]
+* link:https://security-tracker.debian.org/tracker/data/json[Debian Security Tracker]
+* link:https://linux.oracle.com/security/oval[Oracle OVAL]
+* link:https://packages.vmware.com/photon/photon_oval_definitions/[Photon OVAL]
+* link:https://support.novell.com/security/oval/[SUSE OVAL]
+* link:https://security-metadata.canonical.com/oval/[Ubuntu OVAL]
+* link:https://nvd.nist.gov/vuln/search[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
++
+[NOTE]
+====
+This product uses the NVD API but is not endorsed or certified by the NVD.
+====
+* link:https://github.com/stackrox/stackrox/blob/master/scanner/updater/manual/vulns.go[StackRox]: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
+* link:https://www.redhat.com/security/data/metrics/repository-to-cpe.json[repository-to-cpe.json]: Maps RPM repositories to their related CPEs, which is required for matching vulnerabilities for RHEL-based images.
+
+[id="scanner-v4-vuln-sources"]
+== Scanner V4 sources
+
+Scanner V4 uses the following vulnerability sources:
+
+https://access.redhat.com/security/data/csaf/v2/vex/[Red{nbsp}Hat VEX]:: Used with release 4.6 and later. This source provides vulnerability data in link:https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#45-profile-5-vex[Vulnerability Exploitability eXchange(VEX)] format. {product-title-short} takes advantage of VEX benefits to significantly shorten the time needed for the initial loading of vulnerability data, and the space needed to store vulnerability data.
++
+Using VEX affects the way that vulnerabilities are shown. Previously, due to the nature of Red Hat's OVAL vulnerability data, {product-title-short} displayed advisories, such as RHSAs, RHBAs, and RHEAs, instead of CVEs for fixed vulnerabilities that affected Red Hat products. With the switch to VEX vulnerability data, {product-title-short} now displays CVEs for all vulnerabilities affecting Red Hat products.
```

Comment:
Let's remove this, actually. Things have changed very last-minute, and we will still show RHSAs and such for now (expected to change in 4.7)